Hudl.Mjolnir/Command/Command.cs

```diff
@@ -380,9 +380,17 @@ private Task<TResult> ExecuteInIsolation(CancellationToken cancellationToken)
                 CircuitBreaker.MarkSuccess(stopwatch.ElapsedMilliseconds);
                 CircuitBreaker.Metrics.MarkCommandSuccess();
             }
-            catch (Exception)
+            catch (Exception e)
             {
-                CircuitBreaker.Metrics.MarkCommandFailure();
+                if (CommandContext.IsExceptionIgnored(e.GetType()))
```

Comment:
Having thought about this a bit more, I think I like the non-inheritance checking way. It's easy to opt into more types if you want parents/children to also be ignored, but if we use `IsAssignableFrom()` and assume the whole hierarchy, it doesn't leave the option to opt out of any children that you don't want ignored.